modules/waypoints_dict_to_kml.py

```diff
@@ -0,0 +1,37 @@
+"""
+Module to convert waypoints dict to waypoints list, and generate kml file.
+"""
+import pathlib
+
+from modules.common.kml.modules.waypoints_to_kml import waypoints_to_kml
+
+
+def waypoints_dict_to_kml(waypoint_name_to_coordinates: "dict[str, tuple[float, float]]", 
+                          document_name: str,
+                          save_path: pathlib.Path) -> bool:
```

Comment:
Make this a conversion from a dictionary of string and tuple to a list of tuple (the required input to `waypoints_to_kml()` ). Don't call the KML function in here.

Return `False, None` if the dictonary is empty.